### PR TITLE
add vscode-org-mode

### PR DESCRIPTION
**Pull request by KlavsKlavsen**
Signed-off-by: Klavs Klavsen <klavs@enableit.dk>

### CODE REVIEW

File: extensions.json
Comment (jankeromnes) [inline extensions.json:1571]: - The `publisher` in the extension's [package.json](https://github.com/vscode-org-mode/vscode-org-mode/blob/e8cc69868af427f5fd03c8ad125b493cbf2e8292/package.json#L7) seems to be `vscode-org-mode`, not `tootone`
- The latest official version is `1.0.0`. I think we don't want to publish `1.1.0-SNAPSHOT` (or whatever state the default branch currently is in), so it's better to publish `1.0.0` specifically, by checking out the relevant tag, branch, or commit

```suggestion
      "id": "vscode-org-mode.org-mode",
      "repository": "https://github.com/vscode-org-mode/vscode-org-mode",
      "version": "1.0.0",
      "checkout": "e8cc69868af427f5fd03c8ad125b493cbf2e8292"
```

**Review by jankeromnes — APPROVED** [review]
Thanks for your contribution @KlavsKlavsen! 👍

I've suggested a few changes, but I can apply them automatically from GitHub before merging, so you don't need to do anything else here. 😎 🌴

If you add more extensions in the future, please use this repository's helper script, e.g. like so:

```
node add-extension https://github.com/vscode-org-mode/vscode-org-mode
```

(Or, even better here,

```
node add-extension https://github.com/vscode-org-mode/vscode-org-mode --checkout=e8cc69868af427f5fd03c8ad125b493cbf2e8292
```
)

**Comment by jankeromnes** [comment]
> but I can apply them automatically from GitHub before merging

Ah, it seems I cannot apply my own suggestion, because I don't have write access to this PR.

So, please apply the suggestion yourself, or please grant maintainers write access to your PR.

**Comment by jankeromnes** [comment]
Thanks!

**Comment by GitMensch** [comment]
Something seems strange here, the changes were applied on 22 Apr but the extension is still not available.
@jankeromnes Do you have an idea what the issue is?

**Comment by jankeromnes** [comment]
@GitMensch Indeed, good catch!

Searching the GitHub Actions logs, I found that auto-publishing fails with this error:

```
Running: git clone --recurse-submodules https://github.com/vscode-org-mode/vscode-org-mode /tmp/repository
Running: git checkout e8cc69868af427f5fd03c8ad125b493cbf2e8292
Running: npm install
[FAIL] Could not process extension: {
  "id": "vscode-org-mode.org-mode",
  "repository": "https://github.com/vscode-org-mode/vscode-org-mode",
  "version": "1.0.0",
  "checkout": "e8cc69868af427f5fd03c8ad125b493cbf2e8292"
}
Error: Command failed: npm install
Error installing vscode.d.ts: Error: Request returned status code: 404
Details: <!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /api/releases/stable</pre>
</body>
</html>

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! org-mode@1.0.0 postinstall: `node ./node_modules/vscode/bin/install`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the org-mode@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by GitMensch** [comment]
There's no release in that repo, just one (outdated) plain git tag. See https://github.com/vscode-org-mode/vscode-org-mode/issues/194 for a request of changing that.

The extension currently to either have to be built from the 1.0 commit [e8cc69868af427f5fd03c8ad125b493cbf2e8292](https://github.com/vscode-org-mode/vscode-org-mode/commit/e8cc69868af427f5fd03c8ad125b493cbf2e8292) or from HEAD as something like 1.1.0-SNAPSHOT-$(date +%FT%T).

**Pull request merged**
